peakorc.py

```diff
@@ -1,6 +1,7 @@
 import falcon
 from peakmodels import *
 from playhouse.shortcuts import model_to_dict
+import numpy as np
```

Comment:
is shortening to np really needed? two letter names make me twitch

Reply:
But I can reach them with one hand! You're absolutely right, I don't think it makes the code any clearer - I'll fix and push the changes.